chore: update templated files

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/9ae5ffde-098d-43fa-9bf1-5a37f69ff67d/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

parthea commented:
Hey @m-strzelczyk ,

Autosynth is backing out the changes that you made in `CONTRIBUTING.rst` because it's an autogenerated file. The template lives [here](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/CONTRIBUTING.rst). Your update looks helpful for all repos, so I've proposed that we update templated `CONTRIBUTING.rst` [here](https://github.com/googleapis/synthtool/issues/1124). PRs are welcome in googleapis/synthtool!

parthea reviewed (changes requested):
#55 modified the autogenerated `CONTRIBUTING.rst` file. The section on `Samples and code snippets` should be added to the templated `CONTRIBUTING.rst` file [here](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/CONTRIBUTING.rst). I've [opened a bug](https://github.com/googleapis/synthtool/issues/1124) to track this work item.

parthea commented:
Hi @m-strzelczyk, Please can you take a look?

m-strzelczyk commented:
> #55 modified the autogenerated `CONTRIBUTING.rst` file. The section on `Samples and code snippets` should be added to the templated `CONTRIBUTING.rst` file [here](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/CONTRIBUTING.rst). I've [opened a bug](https://github.com/googleapis/synthtool/issues/1124) to track this work item.

I have merged that change and regenerated the PR, it should not remove that part from CONTRIBUTING.rst once it's done. I'll check back soon to see if this worked.

yoshi-automation committed:
changes without context

        autosynth cannot find the source of changes triggered by earlier changes in this
        repository, or by version upgrades to tools such as linters.
